### DISCUSSION TITLE
vs code 擴充套件 git graph

### DISCUSSION THREAD

**Discussion by danwan1027** [1101-GS4719-C/Python-Community-GS4719-C#720] [General]
多人協作時，git 的版本及分之容易混亂到不行， vs code 提供 git graph 的套件，讓混亂的分支及版本可以有圖像化的樣子
如圖
[
![](https://ardalis.com/static/aac4622827ac44710e87e0d26a1df06f/a6d36/image-git-graph.png)
](url)

附上官方文件說明：https://marketplace.visualstudio.com/items?itemName=mhutchie.git-graph

**Comment by danwan1027** [comment]
大家快去下載

**Comment by Yunhans** [comment]
Github多人協作到後來真的超亂

**Comment by williamyen0623** [comment]
git graph真的超好用！大推！
